docs/ops.html

```diff
@@ -262,6 +262,40 @@ <h4 class="anchor-heading"><a id="basic_ops_share_group" class="anchor-link"></a
     </li>
   </ul>
 
+  <p>To reset the offsets of a share group, use the "--reset-offsets" option:
+
+  <p>
+    It has 2 execution options:
+  <ul>
+    <li>
+      --dry-run: to display which offsets to reset.
+    </li>
+    <li>
+      --execute : to execute --reset-offsets process.
+    </li>
+  </ul>
+
+  <p>
+    --reset-offsets also has the following scenarios to choose from (at least one scenario must be selected):
```

Comment:
Isn't it that it has to be only one scenario not `at least`? Currently help displays following:

```
You must choose one of the following
                                         reset specifications: --to-datetime,
                                         --to-earliest, --to-latest.